metis-indexing/src/main/java/eu/europeana/indexing/mongo/TimespanUpdater.java

```diff
@@ -23,5 +23,6 @@ protected void update(MongoPropertyUpdater<TimespanImpl> propertyUpdater) {
     propertyUpdater.updateMap("isPartOf", TimespanImpl::getIsPartOf);
     propertyUpdater.updateMap("dctermsHasPart", TimespanImpl::getDctermsHasPart);
     propertyUpdater.updateArray("owlSameAs", TimespanImpl::getOwlSameAs);
+    propertyUpdater.updateMap("notation", TimespanImpl::getSkosNotation);
```

Comment:
In order to get the skos notation from `TimespanImpl` you will first need to set it. See `TimespanFieldInput`.

Reply:
I see. Updated the ```TimespanFieldInput``` class